misk-aws/src/test/kotlin/misk/jobqueue/sqs/DockerSqs.kt

```diff
@@ -43,12 +44,14 @@ internal object DockerSqs : ExternalDependency {
       // NB(mmihic): Because the client port is embedded directly into the queue URLs, we have to use
       // the same external port as we do for the internal port
       val exposedClientPort = ExposedPort.tcp(clientPort)
-      withImage("pafortin/goaws")
+      withImage("softwaremill/elasticmq:1.4.2")
         .withName("sqs")
         .withCmd(listOf("goaws"))
```

Comment:
Is this command still the same?

Reply:
Thanks, removed. Their entry point seems to ignore arguments which is why this wasn't causing an issue